recipes/scikit-bio/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "0.4.2" %}
+
+package:
+  name: scikit-bio
+  version: {{ version }}
+
+source:
+  fn: scikit-bio-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-bio/scikit-bio-{{ version }}.tar.gz
+  md5: bbbe704410c59083e9eb65fa9d01a50d
+
+build:
+    number: 0
```

Comment:
Thanks @jakirkham, we have added this to skip windows, but we saw this
message after submitting the commit, so it will build again on appveyor,
sorry!

On (Jul-17-16|10:19), jakirkham wrote:
> > @@ -0,0 +1,91 @@
> > +{% set version = "0.4.2" %}
> > +
> > +package:
> > +  name: scikit-bio
> > +  version: {{ version }}
> > +
> > +source:
> > +  fn: scikit-bio-{{ version }}.tar.gz
> > +  url: https://pypi.io/packages/source/s/scikit-bio/scikit-bio-{{ version }}.tar.gz
> > +  md5: bbbe704410c59083e9eb65fa9d01a50d
> > +
> > +build:
> > +    number: 0
> 
> Please add the following to `build`.
> 
> ```
>   skip: true  # [win]
> ```
> 
> Also please feel free to include `[skip appveyor]` in subsequent commits to stop that CI from building.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/conda-forge/staged-recipes/pull/1071/files/f943385d0ec817e7236b1cc757dddb4f8c4744d2#r71085848